plugins/inventory-backend/src/inventory/AggregatorInventory.ts

```diff
@@ -19,14 +19,14 @@ import { Component, Inventory } from './types';
 export class AggregatorInventory implements Inventory {
   inventories: Inventory[] = [];
 
-  list(): Promise<Array<Component>> {
-    return Promise.all(this.inventories.map(i => i.list())).then(lists =>
+  list(): Promise<Component[]> {
+    return Promise.all(this.inventories.map((i) => i.list())).then((lists) =>
```

Comment:
since then was this preferred? 🤔

Reply:
https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-arrowparens-to-always-7430httpsgithubcomprettierprettierpull7430-by-kachkaevhttpsgithubcomkachkaev

I'm ok with it x)